editor/src/main/com/mbrlabs/mundus/tools/ToolManager.java

```diff
@@ -93,8 +93,11 @@ public void deactivateTool() {
     }
 
     public void setDefaultTool() {
-        deactivateTool();
-        activateTool(translateTool);
+        if (activeTool == null)
```

Comment:
Should be: `activeTool == null || activeTool == modelPlacementTool`
because if you are placing models and then hit ESC, the modelPlacementTool will be reset but still is active.
That's good for all the other tools, but not in that case...you would have to select a new tool manually